### PR TITLE
documentation: add docs for Sagemaker Model Parallel 1.3, released with PT 1.8

### PR DESCRIPTION
**Pull request by rahul003**
*Issue #, if available:*

*Description of changes:*
PT 1.8 container coming soon has SMP 1.3. Adding docs for that. 
*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [x] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [x] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [x] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [x] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: doc/api/training/smd_model_parallel_general.rst [hunk 1]
```diff
@@ -304,6 +303,45 @@ table are optional.
    |                   |                         |                 | provided by                       |
    |                   |                         |                 | SageMaker.                        |
    +-------------------+-------------------------+-----------------+-----------------------------------+
+   | ``active_microbatches`` | int          |  ``partitions`` + 2          | The number of                      |
```
Comment (rahul003) [inline doc/api/training/smd_model_parallel_general.rst:306]: Fixing

File: doc/api/training/smd_model_parallel_general.rst [hunk 2]
```diff
@@ -363,25 +389,25 @@ since there are 8 GPUs in a ``p3dn.24xlarge`` instance.
...
-previous example, the ``mp_rank`` of process 1 is 0, and ``mp_rank`` of
+``mp_rank`` is the rank of a process within its own ``mp_group``. In the
+previous example, the ``mp_rank`` of process 1 is 0, and ``mp_rank`` of
 process 6 is 1.
 
 Analogously, the library defines ``dp_group``\ s as the sets of processes that
 all hold the same model partition, and perform data parallelism among
 each other. In the example above, there are two ``dp_group``\ s,
-``[0, 1, 2, 3]`` and ``[4, 5, 6, 7]``,
+``[0, 1, 2, 3]`` and ``[4, 5, 6, 7]``,
```
Comment (rahul003) [inline doc/api/training/smd_model_parallel_general.rst:407]: Not sure why this happened, Talia ran this through an rst formatter

**Comment by rahul003** [comment]
@ChoiByungWook Would it be possible to get this in before the next release?

**Review by TEChopra1000 — DISMISSED** [review]

**Review by ahsan-z-khan — APPROVED** [review]

**Comment by rahul003** [comment]
Could you please help with this error: `AttributeError: 'ServiceManager' object has no attribute '_tfs_grpc_port_range'`

**Review by TEChopra1000 — APPROVED** [review]

**Pull request merged**